### PR TITLE
Add drone outline for Arenda in OpenAerialMap

### PR DESCRIPTION
**Pull request by maning**
Adding a drone imagery we created for Lupang Arenda, Taytay, Philippines.  We want to make this default or at least the best source for this area. 

See context of the field collection here: https://github.com/OSMPH/papercut_fix/issues/54

### CODE REVIEW

**Comment by maning** [comment]
Tests are failing, per FAQ, I used geojson.io to create the geojson then added the required properties.  Can someone point to the issue I should fix.

**Comment by grischard** [comment]
Thank you! You have a FeatureCollection where you should have one Feature, maybe with multiple rings. I'm sure that if you look at other source files to see how they do it, you'll be able to fix it yourself.

**Comment by maning** [comment]
Still getting errors on the invalid license (CC-BY 4.0).  I used this license referred to OAM's legal [page](https://openaerialmap.org/legal/)

Screenshot 2019-07-11 17 53 34

File: sources/asia/ph/Arenda-OAM.geojson [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      "type": "Feature",
+      "properties": {
+        "id": "Arenda_OAM",
+        "name": "Lupang Arenda, Taytay Drone Imagery",
+        "type": "tms",
+        "url": "https://tiles.openaerialmap.org/5d25d7bf161a790005c03e6e/0/5d25d7bf161a790005c03e6f/{zoom}/{x}/{y}.png",
+        "min_zoom": 16,
+        "max_zoom": 22,
+        "country_code": "PH",
+        "license": "CC-BY 4.0",
```
Comment (andrewharvey) [inline sources/asia/ph/Arenda-OAM.geojson]: Per https://github.com/osmlab/editor-layer-index/blob/ec3561b31f161cbf5452fdd9edf12f89b1f05901/schema.json#L68-L70 this must be a [SPDX identifier](https://spdx.org/licenses/). so should be `CC-BY-4.0`. You're missing a `-`. That's why the check is failing.

**Review by andrewharvey — CHANGES_REQUESTED** [review]
looks good to me, just a minor formatting issue

File: sources/asia/ph/Arenda-OAM.geojson (hunk 1 above)
Comment (maning) [reply sources/asia/ph/Arenda-OAM.geojson]: Oh thank you!

**Pull request merged**